webhook_incoming/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 Hoang Tran <thhoang.tr@gmail.com>.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Webhook Incoming",
+    "summary": "Receive incoming requests from webhooks",
+    "version": "16.0.0.0.1",
+    "author": "Hoang Tran,Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "website": "https://github.com/OCA/webhook",
+    "depends": ["base", "base_automation", "webhook_base", "queue_job"],
```

Comment:
"webhook_base" already depends on "base_automation" so no need to add "base" and "base_automation" here